k8s/mongodb/mongo-data-configdb-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+##########################################################
+# This YAML file desribes a k8s pvc for mongodb configDB #
+##########################################################
+
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mongo-configdb-claim
+  annotations:
+    volume.beta.kubernetes.io/storage-class: slow-configdb
+spec:
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
The quoted sentence only says that Azure Disk supports deletion. It doesn't say if Azure Disk supports `Retain`. (It says nothing about support or lack of support.)

Anyway, if `Retain` works on Azure, then keep it in: that's what we want.